sympy/printing/str.py

```diff
@@ -140,6 +140,11 @@ def _print_Exp1(self, expr):
     def _print_ExprCondPair(self, expr):
         return '(%s, %s)' % (expr.expr, expr.cond)
 
+    def _print_Piecewise(self, expr):
+        return 'Piecewise(%s)' % ', '.join([self._print(a)
```

Comment:
I'm trying to comma-join the strings of the ExprCondPairs and the string for the default expression. Apparently this works because it is printing properly, but something is awry somewhere because of the new XFAIL marked above.